Makefile

```diff
@@ -1,41 +0,0 @@
-include .env
-export
-
-CHANGED_FILES := $(shell git ls-files --modified --other --exclude-standard)
-CHANGED_FILES_IN_BRANCH := $(shell git diff --name-only $(shell git merge-base origin/main HEAD))
-
-.PHONY : install_deps setup pre-commit pre-commit-in-branch pre-commit-all test help
-
-install_deps:  ## Install all dependencies.
-	pip install -r dev-requirements.txt
-	pip install -e .
```

Comment:
be aware that this is used for the functional tests

Reply:
Agreed, the functional test workflows will be updated to use the `hatch` scripts that do the same.

Reply:
I'm not sure if workflow runs are viewable, but if you can see [this](https://github.com/dbt-labs/dbt-athena/actions/runs/11276425220/job/31360167516), I got functional tests to work with `hatch`.

Reply:
All good here, you addressed my concerns on this https://github.com/dbt-labs/dbt-athena/pull/732#discussion_r1793927386